src/date-range-picker/calendar/index.tsx

```diff
@@ -275,13 +274,9 @@ function Calendar(
     setEndDateString(e.detail.value);
   };
 
-  let constrainttextId = useUniqueId('awsui-area-date-range-picker');
-  constrainttextId = i18nStrings.dateTimeConstraintText ? constrainttextId : '';
-
   return (
     <>
-      <InternalSpaceBetween size="m">
-        {/* The application role is necessary for screen-readers to allow arrow navigation by default. */}
+      <InternalSpaceBetween size="s">
```

Comment:
Adjusted spacing to account for a 4px top padding of the form field that is now used to provide the constraint text.